wrappers/c/zxing.cpp

```diff
@@ -0,0 +1,202 @@
+#include "zxing.h"
+
+#include <ZXing/ImageView.h>
+#include <ZXing/BarcodeFormat.h>
+#include <ZXing/DecodeHints.h>
+#include <ZXing/Result.h>
+#include <ZXing/ReadBarcode.h>
+
+#ifdef __cplusplus
```

Comment:
The #ifdef here is unnecessary as this file is only every compiled by a c++ compiler.